CHANGELOG.md

```diff
@@ -8,14 +8,54 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Added
 
+- [Codelists] Implement the `complete` attribute. [#45]
+- [Codelists] Codes may have equality compared with strings - the `value` of a Code is compared. [#45]
+- [Codelists] Add v2.02 Codelist mapping file. [#45]
+
+- [Exceptions] Add an `error_log` attribute to ValidationErrors. [#45]
+
+- [Resources] Add method to load data files relating to pyIATI (rather than the IATI Standard, or tests). [#45]
+- [Resources] Allow test files to be located within sub-folders by including slashes (`/`) in the name. [#45]
+- [Resources] Detect encoding of files that are not UTF-8. [#45]
+
+- [Validation] Change from `validate.py` to `validator.py` to improve readability of code using this module. [#86]
+- [Validation] Add a YAML file containing error information. [#117]
+- [Validation] Check whether a string is valid XML - truthy. [#45]
+- [Validation] Check whether a string is valid XML - detailed error information. [#45]
+- [Validation] Provide custom error messages for lxml errors stating that a string is not XML caused by published IATI data. [#90]
+- [Validation] Check whether a Dataset is valid against an IATI Schema - truthy. [#45]
+- [Validation] Check whether a Dataset is valid against an IATI Schema - detailed error information. [#45]
+- [Validation] Provide custom error messages for lxml errors stating that a Dataset is not IATI XML caused by published IATI data. [#92]
```

Comment:
Can this be rephrased? It's hard to understand atm. Something like: `Provide custom error messages when lxml returns errors for data that is not IATI XML in published IATI data.`

Reply:
https://github.com/IATI/pyIATI/pull/209/commits/6b61368da70b82f1ffc74ecb2987c93f481c4447